test/bdd/fixtures/vc-rest/docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
     environment:
       - VC_REST_HOST_URL=${VC_HOST}:${VC_PORT}
       - EDV_REST_HOST_URL=${EDV_HOST}:${EDV_PORT}
+      - SIDETREE_HOST_URL=http://localhost:48326/document
```

Comment:
may be we can move this config to .env file.

Reply:
next PR